packages/loot-core/src/client/actions/account.ts

```diff
@@ -227,12 +227,3 @@ export function markAccountRead(accountId) {
     accountId: accountId,
   };
 }
-
-export function getBanks() {
```

Comment:
Now that `send` is typed, could not find `banks` handler and seems like this is not used anymore